src/components/Articles/index.tsx

```diff
@@ -0,0 +1,76 @@
+'use client'
+import './styles.scss';
+import image from "./imagen/Image.jpg";
+import Image from "next/image";
+import React, { useState, useEffect } from 'react';
+import { getJsonData } from '../../app/Async/servicerequest';
+import BtnComponent from "@/components/Articles/components/buttoncomponent";
+import SlideNavigation from "@/components/Articles/components/slidecomponent";
+import ButtonComponent from '@/components/Articles/components/buttoncomponent';
+import Slide from '@/components/Articles/components/slidecomponent';
+
+interface UserData {
+  postId: number;
+  id: number;
+  name: string;
+  email: string;
+  body: string;
+}
+const SliderComponent: React.FC = () => {
+  const [userData, setUserData] = useState<UserData[]>([]);
+  const [currentSlide, setCurrentSlide] = useState(0);
+  useEffect(() => {
+    const fetchUserData = async () => {
+      try {
+        const url = 'https://jsonplaceholder.typicode.com/comments';
```

Comment:
There is an error this data should come from an env vars. example 
```
process.env.REACT_APP_BASE_API
```
and you can use something like 
```
 const url = `${this.process.env.REACT_APP_BASE_URL}/comments`;
```